frontend/public/locales/fr/apply-adult.json

```
@@ -422,8 +422,8 @@
   },
   "living-independently": {
     "page-title": "Vivre de manière indépendante",
-    "description": "Vivre indépendamment de vos parents signifie que vous avez 16 ou 17 ans, que vous ne recevez aucune aide financière de vos parents et que votre résidence principale est différente de la leur.",
-    "form-instructions": "Vivez-vous indépendamment de vos parents ?",
+    "description": "Vivre de manière indépendante signifie que vous avez 16 ou 17 ans, que vous ne recevez pas d'aide financière de vos parents et que votre résidence principale est différente de la leur.",
+    "form-instructions": "Vivez-vous indépendamment de vos parents\u00a0?",
```

There is not supposed to have a space before question mark in French. 

```suggestion
    "form-instructions": "Vivez-vous indépendamment de vos parents?",
```